pod/perldata.pod

```diff
@@ -520,13 +522,94 @@ C<$who::0>, and a C<$who's> variable.  The last two would be the
 $0 and the $s variables in the (presumably) non-existent package
 C<who>.
 
-In fact, a simple identifier within such curlies is forced to be
-a string, and likewise within a hash subscript.  Neither need
-quoting.  Our earlier example, C<$days{'Feb'}> can be written as
-C<$days{Feb}> and the quotes will be assumed automatically.  But
-anything more complicated in the subscript will be interpreted as an
-expression.  This means for example that C<$version{2.0}++> is
-equivalent to C<$version{2}++>, not to C<$version{'2.0'}++>.
+In fact, a simple identifier within such curly braces is forced to be a
+string, and likewise within a hash subscript. Neither need quoting. Our
+earlier example, C<$days{'Feb'}> can be written as C<$days{Feb}> and the
+quotes will be assumed automatically. But anything more complicated in
+the subscript will be interpreted as an expression. This means for
+example that C<$version{2.0}++> is equivalent to C<$version{2}++>, not
+to C<$version{'2.0'}++>.
+
+There is a similar problem with interpolation with text that looks
+like an array subscript or hash subscript. Placing a simple variable
+like C<$who> immediately in front of text like C<"[1]"> or C<{foo}>
+would cause the variable to be interpolated as a lookup into C<@who> or
+C<%who>, and again using braces will prevent this from happening.
+
+    $who = "Larry Wall";
+    print "${who}[1] is the father of the Perl programming language\n";
+
+will be treated the same as
+
+    $who = "Larry Wall";
+    print $who, "[1] is the father of the Perl programming language\n";
+
+but
+
+    $who = "Larry Wall";
+    print "$who[1] is the father of the Perl programming language\n";
+
+would attempt to lookup index 1 in the C<@who> array.
```

Comment:
"lookup" is only a noun; the verb is "look up".

Reply:
I ditched lookup/"look up" entirely and replaced it with access. Changes pushed. Thanks!